### ISSUE TITLE
The target spacing (minimum) Understanding document has room for improvements

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3473]
The Understanding document could use a quickly review to improve readability. Nothing major, but a few sections forced me to re-read. This sentence about the [Inline exception](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#intent:~:text=When%20multiple%20links,many%20design%20contexts.) in the Intent>Exceptions section, in particular stood out:
> When multiple links are embedded in blocks of texts in smaller text sizes, guaranteeing that undersized links in adjacent lines of text fulfill the spacing exception (their 24 CSS pixel diameter circle don't intersect any other links or their circles) would require a large line height which can be undesirable in many design contexts.

It is a _very_ long sentence, within a larger paragraph. It is very difficult to make sense of without rereading. It can be broken up into simpler sentences.

---

There are also a number of places where the addition of commas would improve readability, [for instance](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#intent:~:text=The%20requirement%20is%20independent%20of%20the%20zoom%20factor%20of%20the%20page%3B%20when%20users%20zoom%20in%20the%20CSS%20pixel%20size%20of%20elements%20does%20not%20change.):
> The requirement is independent of the zoom factor of the page; when users zoom in the CSS pixel size of elements does not change.

---

The understanding document also seems to be missing any additional guidance on the topic of the [first Note](https://www.w3.org/TR/WCAG22/#target-size-minimum:~:text=information%20being%20conveyed.-,NOTE%201,gradient%20of%20colors%2C%20or%20editable%20areas%20where%20you%20position%20the%20cursor.,-NOTE%202) in the normative document:
> Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders, color pickers displaying a gradient of colors, or editable areas where you position the cursor.

For that matter, the user agent exception is also at best sparse in the Understanding document (and I suspect something like a color picker may actually traverse the note and UA exception)

**Comment by mbgower** [comment]
@ljoakley you okay taking this one one?
